ehitchhiking-api/src/main/java/com/exadel/ehitchhiking/controllers/TripsController.java

```diff
@@ -51,8 +51,8 @@ public Response getActive(int id) {
         try {
             list = trips.getAllActive(id);
         } catch (Exception e) {
-            return Response.setError("error");
+            return Response.setError("An error has occurred while retrieving the active!");
```

Comment:
An error has occurred while retrieving the active trips

Reply:
fixed